pkgs/tools/misc/ugtrain/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, autoconf, automake, binutils, glib, libtool, pkg-config, scanmem, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  version = "0.4.1";
+  pname = "ugtrain";
+
+  src = fetchFromGitHub {
+    owner  = "ugtrain";
+    repo   = "ugtrain";
+    rev    = "v${version}";
+    sha256 = "0pw9lm8y83mda7x39874ax2147818h1wcibi83pd2x4rp1hjbkkn";
+  };
+
+  nativeBuildInputs = [ autoconf automake libtool scanmem ];
```

Comment:
Use `autoreconfHook`, instead of both `autoconf` and `automake` (and I think `libtool` as well).